include/circt/Dialect/FIRRTL/FIRRTLOpInterfaces.td

```diff
@@ -275,4 +275,36 @@ def FConnectLike : OpInterface<"FConnectLike"> {
     InterfaceMethod<"Return a source of connection.",
     "Value", "src", (ins)>,
   ];
-}
\ No newline at end of file
+}
+
+def InnerSymbol : OpInterface<"InnerSymbolOpInterface"> {
+  let description = [{
+    This interface describes an operation that may define an
+    `inner_sym`.  An `inner_sym` operation resides 
+    in arbitrarily-nested regions of a region that define a
+    `InnerSymbolTable`.
+    Inner Symbols are different from normal symbols due to 
+    MLIR symbol table resolution rules.  Specifically normal
+    symbols are resolved by first going up to the closest
+    parent symbol table and resolving from there (recursing
+    down for complex symbol paths).  In FIRRTL and SV, modules
+    define a symbol in a circuit or std.module symbol table.
+    For instances to be able to resolve the modules they
+    instantiate, the symbol use in an instance must resolve 
+    in the top-level symbol table.  If a module were a
+    symbol table, modules would try to resolve from their
```

Comment:
"If a module were a symbol table, modules would try", maybe something like:
"If a module was a symbol table, symbol references within the module would try"?